openfe/setup/methods/openmm/equil_rbfe_methods.py

```diff
@@ -668,7 +672,7 @@ def _gather(
         data = []
         for rep in sorted(repeats.items()):
             # then sort within a repeat according to generation
-            nc_files = [ncpath for gen, ncpath in sorted(rep)]
+            nc_files = [ncpath for gen, ncpath in sorted(rep[1])]
```

Comment:
`repeats.items()` yields a list of `[repeat_id, [gen, ncpath]]`, so you need to call `rep[1]` here to be able to sort & unpack.